providers/teams/pages/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
why is this here?

Reply:
Historical reasons once upon a time react used to import some globals, not sure it's still relevant but my IDE still bitches about it. If it's not relevant you could remove it in your tickets on teams provider.